transform/runner_test.go

```diff
@@ -223,14 +270,23 @@ func TestRunnerRun(t *testing.T) {
 			}
 			// Two Bytes tells us that it is an empty list
 			if len(c.IgnoredPatchesString) != 0 || len(response.IgnoredPatches) > 2 {
-				p, err := jsonpatch.DecodePatch([]byte(c.IgnoredPatchesString))
+				ignoredPluginOperations := []PluginOperation{}
+				err := json.Unmarshal(response.IgnoredPatches, &ignoredPluginOperations)
 				if err != nil {
 					t.Error(err)
 				}
-				p2, err := jsonpatch.DecodePatch(response.IgnoredPatches)
+				expectedIgnoredPluginOperations := []PluginOperation{}
+				err = json.Unmarshal([]byte(c.IgnoredPatchesString), &expectedIgnoredPluginOperations)
 				if err != nil {
 					t.Error(err)
 				}
+				p := PatchesFromPluginOperations(expectedIgnoredPluginOperations)
+				p2 := PatchesFromPluginOperations(ignoredPluginOperations)
+				expectedPlugins := PluginNamesFromPluginOperations(expectedIgnoredPluginOperations)
+				foundPlugins := PluginNamesFromPluginOperations(ignoredPluginOperations)
+				if !reflect.DeepEqual(expectedPlugins, foundPlugins) {
```

Comment:
This part isn't ideal. I'm currently comparing the patch contents separately from the plugin list. It may not matter so much in the context of these tests, but if there was more than one rejected patch from different plugins, then swapping the plugin names in the output would still result in a passed test. It might be better to write an `Equals()` method for `[]PluginOperation that compares `PluginName` element-by-element, and calls `ijsonpatch.EqualOperation()` for each `PluginOperation.Operation`.

Reply:
I've addressed this in the latest update to the PR.